src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -455,6 +464,9 @@ public enum SocketOptionName
         SendBuffer = 4097,
         SendLowWater = 4099,
         SendTimeout = 4101,
+        TcpKeepAliveRetryCount = 16,
+        TcpKeepAliveTime = 3,
+        TcpKeepAliveInterval = 17,
```

Comment:
These numbers conflict with DropSourceMembership, TypeOfService, and BlockSource, respectively.

Reply:
Since ````SocketOptionName```` values are traditionally operating into their own namespaces (aka levels), I proposed that we stick to the tradition of reusing native Windows values for options here, explaining the values ````16````, ````3```` and ````17````.

As seen in the source for [SocketOptionName](https://github.com/dotnet/corefx/blob/master/src/System.Net.Sockets/src/System/Net/Sockets/SocketOptionName.cs), there are already many conflicting enumeration values, which is supposed to be fine, since those operate at a different level.

Reply:
If these are all the Windows values, then sounds good.